input/vulnerabilities/RageAgainstTheCage_adb.json

```diff
@@ -47,5 +47,11 @@
       "component" : "platform/system/core",
       "commit" : "dd7bc3319deb2b77c5d07a51b7d6cd7e11b5beb0"
     }
-  }
+  },
+  "Surface": ["local", "system-call", "app"],
```

Comment:
interesting question as to whether this is a "system-call" bug as it is not the system-calls being used to do the exploit but the failure to check the return value of a system-call by a daemon. Perhaps that does count.